### PR TITLE
chore: Update test and related docs for plugin name standard

### PR DESCRIPTION
**Pull request by wilsonwu**
Checklist:

* [x] #2727 Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-rollouts/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this is a chore.
* [x] The title of the PR is (a) [conventional](https://www.conventionalcommits.org/en/v1.0.0/) with a list of types and scopes found [here](https://github.com/argoproj/argo-rollouts/blob/master/.github/workflows/pr-title-check.yml), (b) states what changed, and (c) suffixes the related issues number. E.g. `"fix(controller): Updates such and such. Fixes #1234"`.  
* [x] I've signed my commits with [DCO](https://github.com/argoproj/argoproj)
* [x] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My builds are green. Try syncing with master if they are not. 
* [x] My organization is added to [USERS.md](https://github.com/argoproj/argo-rollouts/blob/master/USERS.md).

### CODE REVIEW

**Comment by wilsonwu** [comment]
Because update the package name, so lint goes wrong, need review.

File: USERS.md [hunk 1]
```diff
@@ -44,3 +44,4 @@ Organizations below are **officially** using Argo Rollouts. Please send a PR wit
 1. [Ubie](https://ubie.life/)
 1. [VISITS Technologies](https://visits.world/en)
 1. [Yotpo](https://www.yotpo.com/)
+1. [DaoCloud](https://daocloud.io)
```
Comment (zachaller) [inline USERS.md:47]: Can this be put in alphabetical order.

Comment (zachaller) [reply USERS.md]: Ended up changing this after looking that the failing tests

**Comment by zachaller** [comment]
Thanks for the PR it is a good idea to update these as well

**Review by zachaller — APPROVED** [review]

File: USERS.md (hunk 1 above)
Comment (wilsonwu) [reply USERS.md]: Sorry for note this late, as this PR merged, I will update it in another one later.

Comment (wilsonwu) [reply USERS.md]: Sorry again, I saw you have done it, thanks so much.

**Pull request merged**